Xamarin.Forms.Platform.UAP/WindowsBasePlatformServices.cs

```diff
@@ -158,6 +161,11 @@ public SizeRequest GetNativeSize(VisualElement view, double widthConstraint, dou
 			return Platform.GetNativeSize(view, widthConstraint, heightConstraint);
 		}
 
+		void UISettingsColorValuesChanged(UISettings sender, object args)
+		{
+			Device.BeginInvokeOnMainThread(() => Application.Current?.OnRequestedThemeChanged(new AppThemeChangedEventArgs(Application.Current.RequestedTheme)));
```

Comment:
Can we change this to just use the native dispatcher?

```C#
this.Dispatcher
```

Reply:
Yes we can!